src/GraphQl/Serializer/ItemNormalizer.php

```diff
@@ -110,8 +110,8 @@ public function normalize(mixed $object, string $format = null, array $context =
      */
     protected function normalizeCollectionOfRelations(ApiProperty $propertyMetadata, iterable $attributeValue, string $resourceClass, ?string $format, array $context): array
     {
-        // to-many are handled directly by the GraphQL resolver
-        return [];
+        // to-many are handled directly by the GraphQL resolver, but also check nested attributes for item query
+        return 'item_query' === $context['graphql_operation_name'] && !\array_key_exists('edges', $context['attributes']) ? [...$attributeValue] : [];
```

Comment:
I find it ugly to check the existence of `edges`, can't we find another solution to do it?

Reply:
I am with you, was also not happy with this approach. I could not find another way without passing other parameters to this method. So I am open for hints. An alternative I first thought creating and passing the `operation` to this method so we can there also check for `operation->getNested()`. Should I try to do it this way?

Reply:
Don't you have the operation in the context? But yes this approach would be better.

Reply:
I found a way to base the check on the operation. This is way more elegant. Thx for your hint.